<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QDialog" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PLSudoku</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="board_layout">
     <item row="0" column="1">
      <widget class="QTableWidget" name="topMiddle">
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QTableWidget" name="topRight">
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="1">
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTableWidget" name="middleLeft">
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>7</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTableWidget" name="middleMiddle">
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTableWidget" name="middleRight">
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>4</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTableWidget" name="topLeft">
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QTableWidget" name="bottomLeft">
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTableWidget" name="bottomMiddle">
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <item row="1" column="0">
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QTableWidget" name="bottomRight">
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout">
     <item>
      <widget class="QPushButton" name="solveBtn">
       <property name="text">
        <string>Solve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetBtn">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
